### PR TITLE
examples/nrf24l01: Replace the hardcode value with enum const

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
Follow https://github.com/apache/incubator-nuttx/pull/6232

## Impact
Minor

## Testing
Need merge together with https://github.com/apache/incubator-nuttx/pull/6232

### CODE REVIEW

**Review by pkarashchenko — COMMENTED** [review]
Please fix "Operator must be next to an operand" in `examples/nrf24l01_term/nrf24l01_term.c` as well

**Review by pkarashchenko — APPROVED** [review]

File: examples/nrf24l01_btle/nrf24l01_btle.c [hunk 1]
```diff
@@ -219,16 +219,16 @@ int nrf24_cfg(int fd)
...
 
   /**************************************************************************
-   * if using RATE_1Mbps from include/nuttx/wireless/nrf24l01.h,
+   * if using RATE_1MBPS from include/nuttx/wireless/nrf24l01.h,
    * tools/checkpatch.sh report error: Mixed case identifier found.
    *
    **************************************************************************/
 
-  nrf24l01_datarate_t datarate = 0; /* RATE_1Mbps */
+  nrf24l01_datarate_t datarate = 0; /* RATE_1MBPS */
```
Comment (pkarashchenko) [inline examples/nrf24l01_btle/nrf24l01_btle.c:227]: @xiaoxiang781216 do you know why `RATE_1MBPS` and `DELAY_1000US` are not used here, but `0` and `3` are used instead?

File: examples/nrf24l01_btle/nrf24l01_btle.c [hunk 2]
```diff
@@ -219,16 +219,16 @@ int nrf24_cfg(int fd)
...
   nrf24l01_retrcfg_t retrcfg =
     {
       .count = 0,
-      .delay = 3 /* DELAY_1000us */
+      .delay = 3 /* DELAY_1000US */
```
Comment (pkarashchenko) [inline examples/nrf24l01_btle/nrf24l01_btle.c:231]: @xiaoxiang781216 let's wipe this out. remove comment and use enum values

File: examples/nrf24l01_btle/nrf24l01_btle.c [hunk 3]
```diff
@@ -219,16 +219,16 @@ int nrf24_cfg(int fd)
   int32_t txpow = DEFAULT_TXPOWER;
 
   /**************************************************************************
-   * if using RATE_1Mbps from include/nuttx/wireless/nrf24l01.h,
+   * if using RATE_1MBPS from include/nuttx/wireless/nrf24l01.h,
```
Comment (pkarashchenko) [inline examples/nrf24l01_btle/nrf24l01_btle.c:222]: please remove the comment as it is not relevant anymore

Comment (xiaoxiang781216) [reply examples/nrf24l01_btle/nrf24l01_btle.c]: Done.

**Review by pkarashchenko — APPROVED** [review]
LGTM!

**Comment by pkarashchenko** [comment]
Do you know why nxstyle is still complaining?

**Comment by xiaoxiang781216** [comment]
I am looking, please wait a moment.

**Comment by xiaoxiang781216** [comment]
Fix here: https://github.com/apache/incubator-nuttx/pull/6226/commits/a07e23faab19ac229febb86dcd9219928d9b5278

**Pull request merged**